packages/test/src/test-worker-activities.ts

```diff
@@ -253,3 +253,27 @@ test('Worker cancels activities after shutdown', async (t) => {
   t.truthy(result?.failed);
   t.true(activityCancelled);
 });
+
+test('Non ApplicationFailure TemporalFailures thrown from Activity are wrapped with ApplicationFailure', async (t) => {
```

Comment:
Should `CancelledFailure`s be wrapped in ApplicationFailures?

Reply:
Yes, unless the activity was requested to be cancelled